Assets/Scripts/UI/BossHealthBar.cs

```diff
@@ -0,0 +1,36 @@
+using FishNet.Object;
+using MarkUlrich.Health;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class BossHealthBar : NetworkBehaviour
+{
+    [SerializeField] private bool _damage;
```

Comment:
What is this bool used for?

Reply:
to test if it works, I will remove this when the other branch of gets merged with the main scene